CHANGELOG.rst

```diff
@@ -1,6 +1,60 @@
 Changelog
 =========
 
+0.6.0 (2018-09-10)
+------------------
+
+MLflow 0.6.0 introduces two major features:
+
+- A Java client API, published to Maven
+- Inherent support for saving and serving SparkML models as MLeap for low-latency serving
+
+In addition to these features, there are a host of improvements and bugfixes to the REST API, Python API, tracking UI, and documentation.
```

Comment:
It might be nice to mention tags above as a bigger feature, and mention some of the other features here (e.g. deleting experiments)